src/components/CreatePost/SelectAsset.tsx

```diff
@@ -30,7 +30,7 @@ export function SelectAssetSuspended({
       }
       options={derivativeAddressesOwned.map((address) => ({
         value: address,
-        label: symbolMap[address] || truncateMiddleIfNeeded(address, 8),
+        label: namesMap[address] || truncateMiddleIfNeeded(address, 8),
```

Comment:
What if the name is extra long? How will it display in the options menu?

Reply:
I added wrapping of it as Matt and Nikita said, ty for catch